src/Persistent/Api.hs

```diff
@@ -126,11 +129,52 @@ makeLenses ''AulaData
 
 deriveSafeCopy 0 'base ''AulaData
 
+
 type AulaLens a = Lens' AulaData a
 type AulaGetter a = Getter AulaData a
 type AulaSetter a = Setter' AulaData a
 type AulaTraversal a = Traversal' AulaData a
 
+
+data DbField a where
+    DbSpaceSet :: DbField (Set IdeaSpace)
+    DbIdeaMap :: DbField Ideas  -- TODO: are these needed? e.g., for removal? or union of two maps?
+    DbIdea :: AUID Idea -> DbField Idea
+    DbUserMap :: DbField Users
+    DbUser :: AUID User -> DbField User
+    DbTopicMap :: DbField Topics
+    DbTopic :: AUID Topic -> DbField Topic
+    DbDelegationMap :: DbField Delegations
+    DbDelegation :: AUID Delegation -> DbField Delegation
+    DbElaborationDuration :: DbField DurationDays
+    DbVoteDuration :: DbField DurationDays
+    DbSchoolQuorum :: DbField Percent
+    DbClassQuorum :: DbField Percent
+    DbLastId :: DbField Integer
+    DbCompleteAulaData :: DbField AulaData
+-- TODO: add more like that, pity it's not composable; or is it?
```

Comment:
`dbIdeaComment` would be a composition `:.:` of `DbIdeaMap`, `At`, `JustF` but these constructors might be not as convenient as their function counterpart so simpler is better here.
(Additionally what gets logged to the changelog will somewhat leak these details)